mage_integrations/mage_integrations/sources/oracledb/README.md

```diff
@@ -0,0 +1,16 @@
+# OracleDB
+
+## Config
+
+You must enter the following credentials when configuring this source:
+
+| Key | Description | Sample value
+| --- | --- | --- |
+| `host` | OracleDB host. | `oracledb.example.com` |
+| `port` | OracleDB exported host. | `1521` (default value) |
+| `service` | 
+Listener service runs on the database server listing for client connection. | `xepdb1` |
```

Comment:
nit: you don't need to change the line here

Reply:
Updated